site/en/models.md

```diff
@@ -164,7 +166,13 @@ Moreover, that variable can easily be overridden during installation.
 However, there are use cases where `system` or even `cwd` should be
 used as location to store resources, hence the `--location` option.
 
+## Downloading on-demand
 
+When you provide a value to a file parameter, such as ocrd_tesserocr's `model`
```

Comment:
```suggestion
When you provide a value to a file parameter, such as ocrd-calamari-recognize's `checkpoint`
```

Reply:
That particular example is not ideal because `checkpoint` still expects a full path with wildcards 😟 But `checkpoint_dir` would be fitting.

Reply:
0c351e45 `s,checkpoint,checkpoint_dir,`